tts.go

```diff
@@ -69,8 +76,12 @@ func (v *tts) Convert(ctx context.Context, voiceID int, text string) (*AudioResp
 	return nil, getApiErr(res)
 }
 
-func (v *tts) StartDuplexStream(ctx context.Context, voiceID int) (*DuplexStream, error) {
-	ws, _, err := websocket.Dial(ctx, fmt.Sprintf("%s/v1/tts/stream/duplex/ws?voiceId=%d", sdkConfig.wsApiUrl, voiceID), nil)
+func (v *tts) StartDuplexStream(ctx context.Context, voiceID int, options *TTSStreamingOptions) (*DuplexStream, error) {
```

Comment:
This is a breaking change. Please change it so the opts param is optional, like so: `, options ...*TTSStreamingOptions)`